### PR TITLE
Handle non-posix local executor

### PR DESCRIPTION
**Pull request by Deixx**

### CODE REVIEW

**Review by Kamoppl — APPROVED** [review]

File: connection/local_executor.py [hunk 1]
```diff
@@ -14,12 +14,15 @@
 
 class LocalExecutor(BaseExecutor):
     def _execute(self, command, timeout):
-        bash_path = TestRun.config.get("bash_path", "/bin/bash")
+        executable_path = TestRun.config.get(
+            "executable_path",
+            "/bin/bash" if os.name == 'posix' else None
+        )
```
Comment (mmichal10) [inline connection/local_executor.py]: For sake of readability, how about
```
default_executable_path = "/bin/bash" if os.name == 'posix' else None
executable_path = TestRun.config.get("executable_path", default_executable_path)
```

Comment (mmichal10) [reply connection/local_executor.py]: BTW the variable `bash_path` is also used in `_rsync()` method. Perhaps we should consider resolving the path in constructor?

Comment (mmichal10) [reply connection/local_executor.py]: Also, `executable_path` -> `cli_path`

Comment (Deixx) [reply connection/local_executor.py]: I suggest that after https://github.com/Open-CAS/test-framework/pull/6 is merged we should remove rsync and replace it with copy* implemented there

**Review by mmichal10 — APPROVED** [review]

File: connection/local_executor.py (hunk 1 above)
Comment (katlapinka) [reply connection/local_executor.py]: https://github.com/Open-CAS/test-framework/pull/6 is merged now, so this could be updated as well :)

**Review by katlapinka — APPROVED** [review]

**Pull request merged**